openedx/core/djangoapps/oauth_dispatch/tests/test_jwt.py

```diff
@@ -102,3 +109,36 @@ def test_create_jwt_for_user(self, user_email_verified, mock_create_roles):
         self.assertDictContainsSubset(additional_claims, token_payload)
         self.assertEqual(user_email_verified, token_payload['email_verified'])
         self.assertEqual(token_payload['roles'], mock_create_roles.return_value)
+
+    def test_default_scopes(self):
+        """
+        Ensure the default scopes are used.
+        """
+        claims = {
+            'tracking_context': {
+                'lms_user_id': self.user.id,
+                'lms_client_id': self.TEST_CLIENT_ID,
+                'lms_ip': '127.0.0.1',
+            }
+        }
+        jwt = jwt_api.create_jwt_for_user(self.user, additional_claims=claims)
+        jwt_default_scopes = jwt_api.create_jwt_for_user(self.user, additional_claims=claims, scopes=self.SCOPES)
+        self.assertEqual(jwt, jwt_default_scopes)
+
+    def test_scopes(self):
+        """
+        Ensure the requested scopes are used.
+        """
+        claims = {
+            'tracking_context': {
+                'lms_user_id': self.user.id,
+                'lms_client_id': self.TEST_CLIENT_ID,
+                'lms_ip': '127.0.0.1',
+            }
+        }
+        scopes = [
+            'user_id',
+        ]
+        jwt = jwt_api.create_jwt_for_user(self.user, additional_claims=claims)
+        jwt_scopes = jwt_api.create_jwt_for_user(self.user, additional_claims=claims, scopes=scopes)
+        self.assertNotEqual(jwt, jwt_scopes)
```

Comment:
Instead or in addition to this assertion, maybe assert that the first doesn’t have user_id and the second does?